client/cypress/e2e/allocationRewardsBox.cy.ts

```diff
@@ -0,0 +1,491 @@
+        cy.get('[data-test=ModalAllocationValuesEdit]').should('not.exist');
+      });
+
+      it('Clicking on `Personal` label or value doesn`t open modal to editing value', () => {
+        cy.get('[data-test=AllocationRewardsBox__section--1]').click();
+        cy.get('[data-test=ModalAllocationValuesEdit]').should('not.exist');
+      });
+
+      it('slider is visible', () => {
+        cy.get('[data-test=AllocationRewardsBox__Slider]').should('be.visible');
+      });
+
+      it('slider thumb exists but isn`t visible', () => {
+        cy.get('[data-test=AllocationRewardsBox__Slider__thumb]').should('exist');
+        cy.get('[data-test=AllocationRewardsBox__Slider__thumb]').should('not.be.visible');
+      });
+
+      it('slider track should be filled in 50%', () => {
+        cy.get('[data-test=AllocationRewardsBox__Slider]').then($sliderEl => {
+          const { width } = $sliderEl[0].getBoundingClientRect();
+
+          cy.get('[data-test=AllocationRewardsBox__Slider__track--0]').should(
+            'have.css',
+            'width',
+            `${width / 2}px`,
+          );
+          cy.get('[data-test=AllocationRewardsBox__Slider__track--0]').should(
+            'have.css',
+            'background-color',
+            'rgb(158, 163, 158)',
+          );
+
+          cy.get('[data-test=AllocationRewardsBox__Slider__track--1]').should(
+            'have.css',
+            'width',
+            `${width / 2}px`,
+          );
+          cy.get('[data-test=AllocationRewardsBox__Slider__track--1]').should(
+            'have.css',
+            'background-color',
+            'rgb(205, 209, 205)',
+          );
+        });
+      });
+
+      it('Clicking on `Personal` label or value doesn`t open modal to editing value', () => {
+        cy.get('[data-test=AllocationRewardsBox__section--1]').click();
+        cy.get('[data-test=ModalAllocationValuesEdit]').should('not.exist');
+      });
+
+      it('Clicking on `Personal` label or value doesn`t open modal to editing value', () => {
+        cy.get('[data-test=AllocationRewardsBox__section--1]').click();
+        cy.get('[data-test=ModalAllocationValuesEdit]').should('not.exist');
+      });
+    },
+  );
+
+  describe(`allocation rewards box (enabled): ${device}`, { viewportHeight, viewportWidth }, () => {
+    before(() => {
+      /**
+       * Global Metamask setup done by Synpress is not always done.
+       * Since Synpress needs to have valid provider to fetch the data from contracts,
+       * setupMetamask is required in each test suite.
+       */
+      cy.setupMetamask();
+    });
+
+    beforeEach(() => {
+      mockCoinPricesServer();
+      localStorage.setItem(IS_ONBOARDING_ALWAYS_VISIBLE, 'false');
+      localStorage.setItem(IS_ONBOARDING_DONE, 'true');
+      visitWithLoader(ROOT_ROUTES.allocation.absolute);
+      cy.intercept('GET', '/rewards/budget/*/epoch/*', { body: { budget: '10000000000' } });
```

Comment:
Instead of budget only I would either:
1. Mock `isDecisionWindowOpen` too.
2. Move the time to when AW is open. I work on that in https://linear.app/golemfoundation/issue/OCT-1481/cy-allocationitem, you can wait and we'll sync.

Eventually (2) will be required, without that we won't test this view end to end.